bower.json

```diff
@@ -13,12 +13,13 @@
   "dependencies": {
     "angular": "1.4.9",
     "angular-sanitize": "1.4.9",
-    "openlayers3": "https://github.com/tombatossals/ol3-compiled.git#3.8.2"
+    "openlayers": "https://github.com/openlayers/ol3/releases/download/v3.16.0/v3.16.0-dist.zip"
```

Comment:
:+1: Sooner or later I'd like to remove bower support and go to npm only as that's the common approach now